Makefile

```
@@ -254,3 +254,13 @@ start-shoot-cert-service:
 		--ignore-operation-annotation=$(IGNORE_OPERATION_ANNOTATION) \
 		--leader-election=$(LEADER_ELECTION) \
 		--config=./controllers/extension-shoot-cert-service/example/00-config.yaml
+
+.PHONY: start-gardener-extensions-validator-aws
```

```suggestion
.PHONY: start-validator-aws
```

(similar to `make start-provider-aws`)